abdm_integrator/hip/serializers/care_contexts.py

```diff
@@ -0,0 +1,30 @@
+from rest_framework import serializers
+
+from abdm_integrator.const import HealthInformationType
+from abdm_integrator.serializers import GatewayCallbackResponseBaseSerializer
+
+
+class LinkCareContextRequestSerializer(serializers.Serializer):
+
+    class PatientSerializer(serializers.Serializer):
+
+        class CareContextSerializer(serializers.Serializer):
+            referenceNumber = serializers.CharField()
+            display = serializers.CharField()
+            hiTypes = serializers.ListField(child=serializers.ChoiceField(choices=HealthInformationType.CHOICES))
+
+        referenceNumber = serializers.CharField()
+        display = serializers.CharField()
+        careContexts = serializers.ListField(child=CareContextSerializer(), min_length=1)
+
+    accessToken = serializers.CharField()
```

Comment:
nit: This is using camelCase

Reply:
This is intentional as request body for ABDM gateway facing APIs are CamelCase.